x-pack/plugins/reporting/server/export_types/csv_searchsource/generate_csv/generate_csv.ts

```diff
@@ -70,20 +70,35 @@ export class CsvGenerator {
       },
     };
 
-    const results = (
-      await lastValueFrom(this.clients.data.search(searchParams, { strategy: ES_SEARCH_STRATEGY }))
-    ).rawResponse as estypes.SearchResponse<unknown>;
+    let results: estypes.SearchResponse<unknown> | undefined;
+    try {
+      results = (
+        await lastValueFrom(
+          this.clients.data.search(searchParams, { strategy: ES_SEARCH_STRATEGY })
+        )
+      ).rawResponse as estypes.SearchResponse<unknown>;
+    } catch (err) {
+      this.logger.error(`CSV export scan error: ${err}`);
+      throw err;
+    }
 
     return results;
   }
 
   private async scroll(scrollId: string, scrollSettings: CsvExportSettings['scroll']) {
-    this.logger.debug(`executing scroll request`);
+    this.logger.info(`Executing scroll request...`);
 
-    return await this.clients.es.asCurrentUser.scroll({
-      scroll: scrollSettings.duration,
-      scroll_id: scrollId,
-    });
+    let results: estypes.SearchResponse<unknown> | undefined;
+    try {
+      results = await this.clients.es.asCurrentUser.scroll({
+        scroll: scrollSettings.duration,
+        scroll_id: scrollId,
+      });
+    } catch (err) {
+      this.logger.error(`CSV export scroll error: ${err}`);
```

Comment:
same problem as above - a timeout error for the search doesn't trigger in a reasonable amount of time